docs/quickstart.rst

```diff
@@ -3,30 +3,210 @@
+
+  .. Note::
+
+    To get started, you'll need to first `install Docker and get it running <https://docs.docker.com/engine/installation/>`_.  The CP Docker Images require Docker version 1.11 or greater.
+
+If you're running on Windows or macOS, you'll need to use `Docker Machine <https://docs.docker.com/machine/install-machine/>`_ to start the Docker host.  Docker runs natively on Linux, so the Docker host will be your local machine if you go that route.  If you are running on Mac or Windows, be sure to allocate at least 4 GB of ram to the Docker Machine.
+
+
+After installing Docker, you can create a Docker machine and begin starting up Confluent Platform.
 
 Installing & Running Docker
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-For this tutorial, we'll run Docker using the Docker client.  If you are interested in information on using Docker Compose to run the images, :ref:`we have docs for that too <quickstart_compose>`.
+For this quickstart, you can use Docker Compose or Docker client.
 
-To get started, you'll need to first `install Docker and get it running <https://docs.docker.com/engine/installation/>`_.  The CP Docker Images require Docker version 1.11 or greater.
+* `Getting Started with Docker Compose`_
+* `Getting Started with Docker Client`_
 
-Running Confluent Platform on Docker
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you're running on Windows or Mac OS X, you'll need to use `Docker Machine <https://docs.docker.com/machine/install-machine/>`_ to start the Docker host.  Docker runs natively on Linux, so the Docker host will be your local machine if you go that route.  If you are running on Mac or Windows, be sure to allocate at least 4 GB of ram to the Docker Machine.
+.. _quickstart_compose:
+
+Getting Started with Docker Compose
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Docker Compose is a powerful tool that enables you to launch multiple Docker images in a coordinated fashion.  It is ideal for platforms like Confluent.  Before you get started, you will need to install both the core `Docker Engine <https://docs.docker.com/engine/installation/>`_ and `Docker Compose <https://docs.docker.com/compose/install/>`_.  Once you've done that, you can follow the steps below to start up the Confluent Platform services.
+
+#. Create and configure the Docker Machine.
+
+   **Important:** If you are using Docker for Mac or Docker for Windows, you can skip this step.
```

Comment:
You may have this backwards -- Linux users don't need docker-machine. I'm not sure why we're recommending it for Mac users either, TBH -- docker-machine is an extra piece that needlessly complicates the Docker experience.

Reply:
Docker for Mac and Windows take care of this as well. So can the step be removed entirely?

Reply:
If we're sure the steps in the quickstart work without it, I think so. We should be sure there aren't other changes needed.

Reply:
i ran through this extensively on mac